docs/table_persistence.md

```diff
@@ -0,0 +1,123 @@
+                                                # UCX Persistence
+
+Enumeration of all UCX persistence elements
+
+## Overview
+Table Utilization:
+
+| Table              | Generate Assessment | Migrate Local Groups | Migrate External Tables | Migrate SQL Warehouses | Upgrade Jobs | Migrate managed tables |
+|--------------------|---------------------|----------------------|-------------------------|------------------------|--------------|------------------------|
+| tables             | RW                  |                      | RO                      |                        |              | RO                     |
+| grants             |                     |                      | RW                      |                        |              | RW                     |
+| mounts             | RW                  |                      | RO                      |                        | RO           | RO                     |
+| permissions        |                     | RW                   |                         |                        |              |                        |
+| jobs               | RW                  |                      |                         |                        | RO           |                        |
+| clusters           | RW                  |
+| external_locations | RW                  |                      | RO                      |
+| workspace          | RW                  | RO                   |                         |                        | RO           |
+
+**RW** - Read/Write the job that generates the table
```

Comment:
most of the things cannot have RW.

Reply:
I'm capturing the Job that generate these.